From b3e4fb81d1740a2ff373b1a77803e829d96317f9 Mon Sep 17 00:00:00 2001
From: Pavan S <psreenivas@habana.ai>
Date: Wed, 18 Sep 2024 09:12:00 +0300
Subject: [PATCH] [SW-202564] common: rename ppboot_cp_preboot function

The ppboot_cp_preboot function is used for coping
preboot, sram bist images hence renaming it to
ppboot_cp_image - no functional change

Signed-off-by: Pavan S <psreenivas@habana.ai>
Change-Id: I37708325ba36e2c3012cbf31b567d6f39ddae599
---

diff --git a/app/ppboot/ppboot.c b/app/ppboot/ppboot.c
index 0075273..b407c94 100644
--- a/app/ppboot/ppboot.c
+++ b/app/ppboot/ppboot.c
@@ -51,7 +51,7 @@
 	preboot_puts_hex_raw(", size: ", desc->img_size, false);
 	preboot_puts(")\n");
 
-	is_valid = ppboot_cp_preboot(desc->loadaddr, desc->spi_offset, desc->img_size, "PREBOOT");
+	is_valid = ppboot_cp_image(desc->loadaddr, desc->spi_offset, desc->img_size, "PREBOOT");
 	if (is_valid && desc->desc_ver > 1) {
 		is_valid = ppboot_is_valid_img_desc(desc);
 		if (!is_valid)
diff --git a/arch/fs1/arc/common.c b/arch/fs1/arc/common.c
index 95caf5c..63f45af 100644
--- a/arch/fs1/arc/common.c
+++ b/arch/fs1/arc/common.c
@@ -422,7 +422,7 @@
 /* TODO: use bootrom code when it is mature enough FSW-2070 */
 #endif /* USE_BOOTROM_SPI_DMA */
 
-bool ppboot_cp_preboot(u64 loadaddr, u32 offset, u32 size, char *name)
+bool ppboot_cp_image(u64 loadaddr, u32 offset, u32 size, char *name)
 {
 	enum dma_error dma_err;
 	enum dma_dest_type dma_type;
diff --git a/arch/gaudi2/arc/common.c b/arch/gaudi2/arc/common.c
index 1c771bd..4b56223 100644
--- a/arch/gaudi2/arc/common.c
+++ b/arch/gaudi2/arc/common.c
@@ -293,7 +293,7 @@
 	image_entry();
 }
 
-bool ppboot_cp_preboot(u64 loadaddr, u32 offset, u32 size, char *name)
+bool ppboot_cp_image(u64 loadaddr, u32 offset, u32 size, char *name)
 {
 	/* Not implemented - ARC BootROM copies and validates preboot */
 	return true;
diff --git a/arch/gaudi2/arm/common.c b/arch/gaudi2/arm/common.c
index 9ca810f..3990446 100644
--- a/arch/gaudi2/arm/common.c
+++ b/arch/gaudi2/arm/common.c
@@ -301,7 +301,7 @@
 	}
 }
 
-bool ppboot_cp_preboot(u64 loadaddr, u32 offset, u32 size, char *name)
+bool ppboot_cp_image(u64 loadaddr, u32 offset, u32 size, char *name)
 {
 	if (name)
 		preboot_puts_raw(name);
diff --git a/arch/gaudi3/arc/asic.c b/arch/gaudi3/arc/asic.c
index 8e0cf64..f8025a6 100644
--- a/arch/gaudi3/arc/asic.c
+++ b/arch/gaudi3/arc/asic.c
@@ -68,8 +68,8 @@
 	preboot_puts_hex_raw(", size: ", desc->img_size, false);
 	preboot_puts(")\n");
 
-	is_valid = ppboot_cp_preboot(desc->loadaddr, desc->spi_offset,
-				     desc->img_size, "SRAM_MBIST_DATA");
+	is_valid = ppboot_cp_image(desc->loadaddr, desc->spi_offset,
+				   desc->img_size, "SRAM_MBIST_DATA");
 
 	if (is_valid && desc->desc_ver > 1) {
 		is_valid = ppboot_is_valid_img_desc_ex((void *)desc);
diff --git a/arch/gaudi3/arc/common.c b/arch/gaudi3/arc/common.c
index e273907..72a8a68 100644
--- a/arch/gaudi3/arc/common.c
+++ b/arch/gaudi3/arc/common.c
@@ -209,7 +209,7 @@
 	return DMA_DEST_HBW;
 }
 
-bool ppboot_cp_preboot(u64 loadaddr, u32 offset, u32 size, char *name)
+bool ppboot_cp_image(u64 loadaddr, u32 offset, u32 size, char *name)
 {
 	u64 dma_dest_addr;
 	enum dma_error dma_err;
diff --git a/include/baremetal.h b/include/baremetal.h
index f4cf564..da9aad7 100644
--- a/include/baremetal.h
+++ b/include/baremetal.h
@@ -207,7 +207,7 @@
 void preboot_cond_arc0_reset(void);
 void preboot_configure_arc(int id, uint64_t startaddr);
 
-bool ppboot_cp_preboot(u64 loadaddr, u32 offset, u32 size, char *name);
+bool ppboot_cp_image(u64 loadaddr, u32 offset, u32 size, char *name);
 void image_start(u64 loadaddr);
 u32 sp_sram_readl(u32 offset);
 void hl_sram_self_repair(void);
